fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
![Codacy](https://app.dev.codacy.org/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function. (wrap-iife)](https://app.dev.codacy.org/manual/devcodacytestuser/CheckPrCommentsPublicProjectDev/pullRequest?prid=19561)